tests/PushNotificationsTests.scala

```diff
@@ -32,15 +32,13 @@ class grtg
     with WskTestHelpers{
   implicit val wskprops = WskProps()
   val wsk = new Wsk()
-  //val credentials = TestUtils.getVCAPcredentials("imfpush")
-  // val appSecret = credentials.get("appSecret");
-  // val url = credentials.get("url");
-  // val appId = url.split("/").last;
+  val credentials = TestUtils.getVCAPcredentials("imfpush")
+  val appSecret = credentials.get("appSecret");
+  val credentialsUrl = credentials.get("url");
+  val appId = credentialsUrl.split("/").last;
+  val url = "https://w3.ibm.com".toJson;
```

Comment:
Please do not use url that are not public. this url is a private url to a company intranet